extend-existing-app-theme-demo/pom.xml

```diff
@@ -0,0 +1,318 @@
+    <modelVersion>4.0.0</modelVersion>
+
+    <name>Reusing Theme jar Demo</name>
+    <groupId>com.vaadin.demo.apptheme</groupId>
+    <artifactId>extend-existing-app-theme-demo</artifactId>
+    <version>1.0-SNAPSHOT</version>
+    <packaging>jar</packaging>
+
+    <properties>
+        <java.version>11</java.version>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+
+        <vaadin.version>19.0-SNAPSHOT</vaadin.version>
+
+        <drivers.dir>${project.basedir}/drivers</drivers.dir>
+        <drivers.downloader.phase>pre-integration-test</drivers.downloader.phase>
+        <flow.version>6.0.0.alpha3</flow.version>
+        <vaadin.spring.version>16.0.0.alpha2</vaadin.spring.version>
+    </properties>
+
+    <parent>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <version>2.4.0</version>
+    </parent>
+
+    <pluginRepositories>
+        <pluginRepository>
+            <id>vaadin-prereleases</id>
+            <url>https://maven.vaadin.com/vaadin-prereleases</url>
+            <!--<url>http://tools.vaadin.com/nexus/content/repositories/vaadin-prereleases</url>-->
+        </pluginRepository>
+        <pluginRepository>
+            <id>central</id>
+            <url>https://repo1.maven.org/maven2/</url>
+            <snapshots><enabled>false</enabled></snapshots>
+        </pluginRepository>
+    </pluginRepositories>
+
+    <repositories>
+        <repository>
+            <id>vaadin-prereleases</id>
+            <url>https://maven.vaadin.com/vaadin-prereleases</url>
+            <!--<url>http://tools.vaadin.com/nexus/content/repositories/vaadin-prereleases</url>-->
+        </repository>
+        <repository>
+            <id>central</id>
+            <url>https://repo1.maven.org/maven2/</url>
+            <snapshots><enabled>false</enabled></snapshots>
+        </repository>
+        <!-- Repository used by many Vaadin add-ons -->
+        <repository>
+            <id>Vaadin Directory</id>
+            <url>https://maven.vaadin.com/vaadin-addons</url>
+            <snapshots><enabled>false</enabled></snapshots>
+        </repository>
+    </repositories>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>com.vaadin</groupId>
+                <artifactId>vaadin-bom</artifactId>
+                <version>${vaadin.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.vaadin.demo.apptheme.reusable</groupId>
+            <artifactId>reusable-theme</artifactId>
+            <version>1.0-SNAPSHOT</version>
+        </dependency>
+        <dependency>
```

Comment:
If you switch to a platform alpha, then you may want to remove flow-server and flow-client deps.

Reply:
Done.